test/v2/controllers/actions/AuthNewEnrolmentOnlyActionSpec.scala

```diff
@@ -42,7 +42,7 @@ class AuthNewEnrolmentOnlyActionSpec extends AnyFreeSpec with Matchers with Mock
 
     def get(): Action[AnyContent] = authAction {
       authedRequest =>
-        Ok(authedRequest.eori)
+        Ok(authedRequest.eoriNumber.value)
```

Comment:
honestly, I'm all for an implicit or an extension so we can just give these anything that extends AnyVal. Or at least the party of me that finds that habit of using AnyVals annoying

Reply:
(please keep using AnyVals, just to be clear)